apps/core/components/sectorDetails/ItinerarySectorDetails.js

```diff
@@ -0,0 +1,52 @@
+// @flow
+
+import * as React from 'react';
+import { Card, StyleSheet } from '@kiwicom/universal-components';
+import { createFragmentContainer, graphql } from '@kiwicom/margarita-relay';
+import {
+  ItineraryTypeRenderer,
+  SectorDetailsWrapper,
+} from '@kiwicom/margarita-components';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+import ItineraryOneWay from '../ItineraryCard/itineraryDetail/ItineraryOneWay';
+import ItineraryReturn from '../ItineraryCard/itineraryDetail/ItineraryReturn';
+// @TODO - gemerated type seems to be broken - import type { ItinerarySectorDetails_data as ItineraryType } from './__generated__/ItinerarySectorDetails_data.graphql';
+
+type Props = {|
+  +data: any, // @TODO - cannot use genereated ?ItineraryType which seems to be broken,
```

Comment:
Not happy about this part but I cannot make it work. If I try to put content directly into `ResultDetailContent` everything works fine, but once I extract it to fragment, flow start to complaining that `$fragmentRefs is missing in object type [1] but exists in $FragmentRef`.

I'll definitely appreciate any tip and maybe some fresh view will help.

Reply:
Change the fragment to 
```
export default createFragmentContainer(ItinerarySectorDetails, {
  data: graphql`
    fragment ItinerarySectorDetails_data on ItineraryInterface {
      __typename
      ...ItineraryOneWay_itinerary
      ...ItineraryReturn_itinerary
    }
  `,
});
```
and it works

Reply:
Thanks, that helped.